Library/Homebrew/keg_relocate.rb

```diff
@@ -19,17 +19,39 @@ def relocate_dynamic_linkage(_old_prefix, _new_prefix, _old_cellar, _new_cellar)
     []
   end
 
-  def relocate_text_files(old_prefix, new_prefix, old_cellar, new_cellar,
-                          old_repository, new_repository)
-    files = text_files | libtool_files
+  def replace_locations_with_placeholders
+    relocate_dynamic_linkage(HOMEBREW_PREFIX.to_s, PREFIX_PLACEHOLDER,
```

Comment:
Tiny nit: might want to make the two prefix args a key/value of a hash or something? Doesn't really matter but I always get a little 😨 about things that rely on arg ordering like this.